.github/workflows/dep-summaries.yml

```diff
@@ -23,7 +23,7 @@ jobs:
       - name: fetch base ref
         id: fetch-base-ref
         if: ${{ github.actor != 'bors-libra' }}
-        uses: ./.github/actions/pr-base-ref
+        uses: libra/actions/pr-base-ref
```

Comment:
we probably want to version this for reproducible builds?

Reply:
Done. This is going to make it extremely tedious to update these as you will need to update the libra/actions code and then search/replace the git sha in this repo.